authors/apps/profiles/tests/base.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import reverse
+from rest_framework.test import APITestCase
+
+
+class BaseTestCase(APITestCase):
+
+    fixtures = ['authors/apps/profiles/tests/fixtures/profile.json', 
+                'authors/apps/profiles/tests/fixtures/user.json']
```

Comment:
@myrdstom Add a new line at the end of this file. Follow this resource to `https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files` to setup your vs code to this automatically.

Reply:
Share this with your colleagues.

Reply:
Done